extensions/Worker.Extensions.Storage.Blobs/src/BlobInputAttribute.cs

```diff
@@ -1,11 +1,13 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using Microsoft.Azure.Functions.Worker.Converters;
 using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
 
 namespace Microsoft.Azure.Functions.Worker
 {
-    [SupportsDeferredBinding]
+    [AllowConverterFallback(true)]
```

Comment:
Same here, should be false?

Reply:
Had a chat with Jacob. For these 2 converters (BlobStorage and Cosmos), both true and false will have same behavior because converter will handle everything (and return fail if Json deserialization throws exception). But for other extensions which will actually use BypassDeferredBinding, it depends how much restrictive they want their converter to be and then decide if they want fallback option or not - something to keep in our extension author guide.

Reply:
@lilieankasem - if you have reason you want this to be `false` I am not against it. Functionally I don't think it will have any impact (at least that I see) to the end user between `false` and `true`.

Reply:
I was thinking `false` for the same reasoning mentioned above, because these two converters are doing everything, fallback is not an option and I felt that false was a better representation of that. Alternatively, if it won't have impact for the scenario where it's all in (like blob and cosmos right now) - then we shouldn't have to provide this attribute at all?